ansible/roles/rancher/tasks/cert-manager.yml

```diff
@@ -5,9 +5,17 @@
   ignore_errors: true
   changed_when: false
   register: crd
-  
+
+- name: Copy cert-manager Custom Resoure Definintion
+  become: true
+  copy: src="{{ item }}" dest="/etc/kubernetes/{{ item }}" mode=600 owner=root
+  with_items:
+    - "cert-manager-crds-{{ cert_manager_version }}.yaml"
```

Comment:
Where is `cert_manager_version` defined?

Reply:
Defined in https://github.com/dominodatalab/ranchhand/blob/master/ansible/roles/rancher/vars/main.yml#L2

>cert_manager_version: v0.9.1